src/redux/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const SET_APP_DATA = "SET_APP_DATA";
 export const SET_ERROR_ACTION = "SET_ERROR_ACTION";
+export const SET_AUTORIZATION_STATUS_TRUE = "SET_AUTORIZATION_STATUS_TRUE";
```

Comment:
weird naming. Better just SET_AUTORIZATION_STATUS = "SET_AUTORIZATION_STATUS"

your action creator function name should explain the rest

Reply:
fixed